src/higley_lab_to_nwb/extractors/processed_imaging_extractor.py

```diff
@@ -0,0 +1,169 @@
+from pathlib import Path
+from typing import Tuple, Literal
+import numpy as np
+from skimage.exposure import rescale_intensity
+from roiextractors.extraction_tools import PathType, DtypeType, get_package
+from roiextractors.imagingextractor import ImagingExtractor
+import h5py
+
+
+from datetime import datetime
+
+
+def _load_data_from_dual_imaging_configuration(file_path: str, process_type: str) -> np.ndarray:
+    """Load pixels matrix from MATLab file generated processing imaging data in dual imaging configuration."""
+
+    with h5py.File(file_path, "r") as file:
+        pixels_matrix = file[process_type][:]
+
+        num_rows, num_columns = file["mask"][:].shape
+
+    return np.nan_to_num(pixels_matrix), num_rows, num_columns
+
+
+def _load_data_from_1p_imaging_configuration(file_path: str, process_type: str) -> np.ndarray:
+    """Load pixels matrix from MATLab file generated processing imaging data in 1p-imaging only configuration."""
+
+    with h5py.File(file_path, "r") as file:
+        pixels_matrix = file["dFoF"][process_type][:]
+
+        num_rows = int(file["R"][:])
+        num_columns = int(file["C"][:])
+
+    return np.nan_to_num(pixels_matrix), num_rows, num_columns
+
+
+def _get_mask_from_dual_imaging_configuration(file_path: str) -> np.ndarray:
+    with h5py.File(file_path, "r") as file:
+        mask = file["mask"][:]
+    return mask
```

Comment:
I would merge this with `_load_data_from_dual_imaging_configuration` to avoid loading the file twice